### PR TITLE
Format in the datepicker

### PR DESCRIPTION
**Pull request by tognit**
Would it be possible to output the format in the data picker based on the company setting?
As well as the language according to company settings

### CODE REVIEW

File: resources/assets/js/components/AkauntingDate.vue [hunk 1]
```diff
@@ -27,6 +27,8 @@
 <script>
 import flatPicker from "vue-flatpickr-component";
 import "flatpickr/dist/flatpickr.css";
+import { de } from 'flatpickr/dist/l10n/de.js';
```
Comment (denisdulici) [inline resources/assets/js/components/AkauntingDate.vue:30]: This shouldn't be loaded by default

File: resources/assets/js/components/AkauntingDate.vue [hunk 2]
```diff
@@ -76,7 +78,8 @@ export default {
 
     data() {
         return {
-            real_model: this.model
+            real_model: this.model,
+            locale: !this.local ? de : en,
```
Comment (denisdulici) [inline resources/assets/js/components/AkauntingDate.vue:82]: What happens if I've set up `fr` in my settings?

**Review by denisdulici — CHANGES_REQUESTED** [review]
Thanks for the contribution, I've added some comments.

**Comment by denisdulici** [comment]
@tognit Any update on this?

**Comment by tognit** [comment]
I'm still there

File: resources/assets/js/components/AkauntingDate.vue (hunk 1 above)
Comment (tognit) [reply resources/assets/js/components/AkauntingDate.vue:30]: The import should be dynamic according to the locale

File: resources/assets/js/components/AkauntingDate.vue (hunk 2 above)
Comment (tognit) [reply resources/assets/js/components/AkauntingDate.vue:82]: in date_group.blade.php you could read out the locale
```javascript
<akaunting-date
.......
      :config="{
            ...........
            locale: '{{ language()->getShortCode() }}',
        }"
..........
></akaunting-date>
```

**Comment by tognit** [comment]
The last update should dynamically adapt the language

**Pull request merged**